src/coreComponents/fieldSpecification/FieldSpecificationManager.cpp

```diff
@@ -182,20 +184,20 @@ void FieldSpecificationManager::validateBoundaryConditions( MeshLevel & mesh ) c
     for( auto const & mapEntry : isTargetSetEmpty )
     {
       GEOS_LOG_RANK_0_IF( ( mapEntry.second == 1 ), // target set is empty
-                          GEOS_FMT( "\nWarning!"
-                                    "\n{}: this FieldSpecification targets (an) empty set(s)"
-                                    "\nIf the simulation does not involve the SurfaceGenerator, check the content of the set `{}` in `{}`. \n",
-                                    fs.getName(), mapEntry.first, fs.getObjectPath() ) );
+                          GEOS_FMT( "\nWarning!\n{0}: this FieldSpecification targets (an) empty set(s)"
+                                    "\nIf the simulation does not involve the SurfaceGenerator, check the content of the set `{1}` in `{2}`. \n",
+                                    fs.getDataContext(), mapEntry.first, fs.getObjectPath() ) );
```

Comment:
Why do you define the indices in the `GEOS_FMT` call?

Reply:
Indeed, it was before I was advised against doing it. I thought it would be clearer.
I remove these ids.